packages/next/src/shared/lib/constants.ts

```diff
@@ -50,9 +50,13 @@ export const TURBOPACK_CLIENT_MIDDLEWARE_MANIFEST =
 export const DEV_CLIENT_MIDDLEWARE_MANIFEST = '_devMiddlewareManifest.json'
 export const REACT_LOADABLE_MANIFEST = 'react-loadable-manifest.json'
 export const SERVER_DIRECTORY = 'server'
+// order based on priority: rare to common
```

Comment:
But why? We didn't used to order rarity since we had .js before .ts and .js is presumably more common.

Reply:
Ah, the comment needed more reasoning. I meant to order `CJS -> MJS -> JS`, and as `.js` is more used, it was up front. But.. IMO `common -> rare` makes more sense for perf.